src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,20 @@
+import styles from "./css/articleCard.module.css"
+
+function ArticleCard({article}){
+    console.log(article.title)
```

Comment:
Rogue console.logs shouldn't be included in commits.

Reply:
The info being delivered via each article card could be made more readable. For example the current date could do with being formatted into something more meaningful and the votes could be labelled.